features/netsocket/TCPServer.h

```diff
@@ -28,7 +28,7 @@
 /** TCP socket server
  *  @addtogroup netsocket
  */
-class TCPServer : public Socket {
+class TCPServer : public InternetSocket {
```

Comment:
Not totally convinced that `TCPServer` needs to exist as a separate class, but I guess it does avoid the implementations of `listen` and `accept` being pulled in unless needed. Would it be more or less efficient to derive from `TCPSocket`, do you think?

Reply:
TCPSocket now support `listen()` and `accept()`. TCPServer deprecated, but left the implementation in place to support old code.